libs/ui-lib/lib/ocm/components/clusterConfiguration/ExternalPlatformDropdown.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { Dropdown, DropdownItem, DropdownToggle, FormGroup } from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { useField } from 'formik';
+import {
+  NUTANIX_CONFIG_LINK,
+  PlatformType,
+  VSPHERE_CONFIG_LINK,
+  getFieldId,
+} from '../../../common';
+
+const INPUT_NAME = 'platform';
+const fieldId = getFieldId(INPUT_NAME, 'input');
+export type ExternalPlatformType = Extract<PlatformType, 'none' | 'nutanix' | 'oci' | 'vsphere'>;
+
+type ExternalPlatformDropdownProps = {
+  isOracleCloudPlatformIntegrationEnabled: boolean;
+  selectedPlatform: ExternalPlatformType;
+  disabledOciTooltipContent: React.ReactNode;
+  isOciDisabled: boolean;
+  onChange: (isOracleOptionSelected: boolean) => void;
+};
+
+export type ExternalPlatformInfo = {
+  label: string;
+  href: string;
+  tooltip: string;
+};
+
+export const externalPlatformTypes: Record<ExternalPlatformType, ExternalPlatformInfo> = {
+  none: {
+    label: 'None',
+    href: '',
+    tooltip: '',
+  },
+  nutanix: {
+    label: 'Nutanix',
+    href: NUTANIX_CONFIG_LINK,
+    tooltip: '',
+  },
+  oci: {
+    label: 'Oracle  (Using custom manifests)',
+    href: 'www.google.es',
+    tooltip:
+      "To integrate with an external partner (for example, Oracle Cloud), you'll need to provide a custom manifest.",
+  },
+  vsphere: {
+    label: 'vSphere',
+    href: VSPHERE_CONFIG_LINK,
+    tooltip: '',
+  },
+};
+
+export const ExternalPlatformDropdown = ({
+  isOracleCloudPlatformIntegrationEnabled,
+  selectedPlatform,
+  disabledOciTooltipContent,
+  isOciDisabled,
+  onChange,
```

Comment:
I am confused by the `isOracleCloudPlatformIntegrationEnabled` and `isOciDisabled` props. It took me a moment to figure out that `isOracleCloudPlatformIntegrationEnabled` is used to filter the OCI option out and `isOciDisabled` is to control if it's enabled/disabled. We should consider different, more descriptive names to make this clearer.

Perhaps `showOCIOption` or `isOciIncluded` instead of `isOracleCloudPlatformIntegrationEnabled`?